### ISSUE TITLE
'placeholderFillStyle' option set with addTiledImage on viewer does not seem to have an effect

### ISSUE THREAD

**Issue by mkoesem** [openseadragon/openseadragon#1283]
I was using version 2.1.0 and the option 'placeholderFillStyle' with addTileImage on viewer. This draws a placeholder while the tile is loading. Now i have updated to version 2.3.0 and the placeholder is not drawn.

I have traced the code and the function `drawTiles` always returns because the array `lastDrawn` is always empty until the tile is loaded. And when the tile is loaded no placeholder is drawn.

How can i fix it? Thanks

**Comment by iangilman** [comment]
Good catch! 

Looks like you can just update that if statement like so:

```
if (tiledImage.opacity === 0 || (lastDrawn.length === 0 && !tiledImage.placeholderFillStyle)) {
```

Would you be up for making a patch to that effect?

**Comment by mkoesem** [comment]
This fix resolves the problem:

    if (tiledImage.opacity === 0 || (lastDrawn.length === 0 && !tiledImage.placeholderFillStyle)) {
        return;
    }
    var tile = lastDrawn[0];
    var useSketch;

    if (tile) {
        useSketch = tiledImage.opacity < 1 ||
            (tiledImage.compositeOperation &&
                tiledImage.compositeOperation !== 'source-over') ||
            (!tiledImage._isBottomItem() && tile._hasTransparencyChannel());
    }

**Comment by mkoesem** [comment]
I am making a pull request

**Comment by iangilman** [comment]
Fixed by #1286
